pandas/io/json.py

```diff
@@ -204,6 +208,12 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
     else:
         json = filepath_or_buffer
 
+    if lines:
+        # If given a json lines file, we break the string into lines, add
+        # commas and put it in a json list to make a valid json object.
+        lines = list(StringIO(json))
+        json = '[' + ','.join(lines) + ']'
```

Comment:
I don't think json is typically unicode, but I think this might break in PY3 if the orginal json i encoded.

Can you see if this is the case? (I am not sure)

```
In [4]: pd.read_json(j.encode('utf-8') + ']')
TypeError: can't concat bytes to str
```